test/unit/test_ansible.py

```diff
@@ -47,7 +46,7 @@ def test_run_ansible_default_values(test_config):
     ansible_access = AnsibleTestAccess()
```

Comment:
I have the feeling at some point in the future we should replace AnsibleTestAccess with a Mock, because nothing other it is implementing at the moment